experimental/hip/CMakeLists.txt

```diff
@@ -8,6 +8,19 @@
 set(IREE_PACKAGE_ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}")
 set(IREE_PACKAGE_ROOT_PREFIX "iree/experimental/hip")
 
+# TODO: maybe merge these variables with the equivalent in the ROCm driver.
+# The obstacle is that they are now experimental and should probably not polute
+# the main CMake file.
+set(IREE_HIP_DRIVER_TARGET_CHIP_DEFAULT "gfx908")
+set(IREE_HIP_DRIVER_TARGET_CHIP "${IREE_HIP_DRIVER_TARGET_CHIP_DEFAULT}" CACHE STRING
+  "Target chip for the HIP driver. This influences conformance tests that need to compile device code. Defaults to \"${IREE_HIP_DRIVER_TARGET_CHIP_DEFAULT}\".")
+set(IREE_HIP_DRIVER_LINK_BC_DEFAULT OFF)
+set(IREE_HIP_DRIVER_LINK_BC ${IREE_HIP_DRIVER_LINK_BC_DEFAULT} CACHE BOOL
+  "Whether to try Linking to AMD Bitcodes. This influences conformance tests that need to compile device code. Defaults to ${IREE_HIP_DRIVER_LINK_BC}.")
+set(IREE_HIP_DRIVER_BC_DIR_DEFAULT "/opt/rocm/amdgcn/bitcode")
+set(IREE_HIP_DRIVER_BC_DIR "${IREE_HIP_DRIVER_BC_DIR_DEFAULT}" CACHE STRING
```

Comment:
we need to fix things so this is not required - we should only require the bc in the compiler - @antiagainst does the new hip driver still need this bc file?
/cc @stellaraccident

Reply:
I will have to check.